README.md

```diff
@@ -30,6 +30,56 @@ revision: HEAD
 
 See `conf/machine` for the complete list of supported devices.
 
+## Quick build
+Please refer to the [Yocto Project Quick Start Guide](https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html) to set up your Yocto Project build environment.
+
+1. Clone this layer and its dependencies
+	```
+	git clone https://github.com/quic-yocto/meta-qcom-hwe.git -b main
+	git clone git://git.yoctoproject.org/meta-qcom -b master
+	git clone git://git.yoctoproject.org/poky -b master
+	```
+
+2. Initialize the build environment:
+	```
+	source poky/oe-init-build-env
+	```
+
+3. Add `meta-qcom` and `meta-qcom-hwe` layers to bblayers.conf:
+	```
+	bitbake-layers add-layer ../meta-qcom
+	bitbake-layers add-layer ../meta-qcom-hwe
+	```
+
+4. Set `MACHINE` in local.conf to one of the supported boards
+	```
+	MACHINE = "qcs6490-rb3gen2-core-kit"
+	```
+
+5. Set `INITRAMFS_IMAGE` in local.conf
+	```
+	INITRAMFS_IMAGE ?= "initramfs-qcom-image"
+	```
+
+6. Build images:
+	```
+	bitbake esp-qcom-image && bitbake core-image-minimal
+	```
+
+	This command generates kernel and rootfs images in the formats `esp-qcom-image-[MACHINE].rootfs-[timestamp].vfat` and `core-image-minimal-[MACHINE].rootfs-[timestamp].ext4`, respectively.
+
+## Quick build with kas
+1. Install kas tool
+	```
+	sudo pip3 install kas
+	```
+2. Build using the KAS configuration for one of the supported boards
+	```
+	kas build meta-qcom-hwe/ci/qcs6490-rb3gen2-core-kit.yml
```

Comment:
Also need the git clone instruction for this layer:

`git clone https://github.com/quic-yocto/meta-qcom-hwe.git -b main`